components/content/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import {Center, View, ViewProps} from 'components/core';
+import React, {useState} from 'react';
+import {ColorValue, GestureResponderEvent, Pressable, Text} from 'react-native';
+import tinycolor from 'tinycolor2';
+import {colorLookup} from 'theme';
+
+export interface ButtonProps extends ViewProps {
+  //  title: string;
+  color?: ColorValue;
+  onPress?: (event: GestureResponderEvent) => void;
+}
+
+export const Button: React.FC<ButtonProps> = ({color = 'color-primary', children, onPress, ...props}) => {
+  const [pressedBgColor] = useState<ColorValue>(tinycolor(colorLookup(color)).setAlpha(0.2).toRgbString());
```

Comment:
`tinycolor` library is used to parse a color string like "#1677FF", apply an opacity and output it as `rgba(r, g, b, opacity)`